### PR TITLE
Rename lib folders

### PR DESCRIPTION
**Pull request by Kriechi**
refs #947

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+testpaths = test
```
Comment (mhils) [inline pytest.ini]: Do we still need to specify `./test` for the runs then?

**Comment by mhils** [comment]
As I mentioned before, I like this change. It seems much more intuitive that mitmproxy provides a package that is actually called mitmproxy, and the same applies to pathod. Technically, both are tools, not libraries (although we obviously want to design the nicely nonetheless).

File: pytest.ini (hunk 1 above)
Comment (Kriechi) [inline pytest.ini]: No, we do not. But since you use this fancy trick to get netlib to test on py3 I'm not sure how to handle it...

Comment (mhils) [inline pytest.ini]: Ok, that is excellent. So we can just keep it for travis but remove it everywhere else?

Comment (cortesi) [inline pytest.ini]: Not if the py.test command is being run from the top of the repo.

**Comment by cortesi** [comment]
This looks good to me. Way back, the reason for the "libmproxy" naming was to distinguish between mitmproxy the tool and mitmproxy the library. Things have evolved over time and this makes less sense nowadays.

**Pull request merged**